aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -448,11 +457,27 @@
     },
     "read": {
       "permissions": [
+        "ec2:DescribeVpcAttribute",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeInternetGateways",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeVpcs",
+        "ec2:DescribeAccountAttributes",
+        "ec2:DescribeSubnets",
         "rds:DescribeDBInstances"
       ]
     },
     "update": {
       "permissions": [
+        "ec2:DescribeVpcAttribute",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeInternetGateways",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeVpcs",
+        "ec2:DescribeAccountAttributes",
+        "ec2:DescribeSubnets",
+        "iam:GetRole",
+        "iam:ListRoles",
         "iam:PassRole",
         "rds:AddRoleToDBInstance",
         "rds:AddTagsToResource",
```

Comment:
In case of enable Performance Insights, Customer will need `pi:*` permissions. Should we add it?